plugins/org.jkiss.dbeaver.ext.gbase8s/src/org/jkiss/dbeaver/ext/gbase8s/edit/GBase8sTableColumnManager.java

```diff
@@ -35,13 +34,25 @@
 import org.jkiss.dbeaver.model.impl.edit.SQLDatabasePersistAction;
 import org.jkiss.dbeaver.model.impl.sql.edit.SQLObjectEditor;
 import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
+import org.jkiss.utils.CommonUtils;
 
 /**
  * @author Chao Tian
  */
 public class GBase8sTableColumnManager extends GenericTableColumnManager
         implements DBEObjectRenamer<GenericTableColumn> {
 
+    @Override
+    protected void addObjectCreateActions(DBRProgressMonitor monitor, DBCExecutionContext executionContext,
+            List<DBEPersistAction> actions,
+            SQLObjectEditor<GenericTableColumn, GenericTableBase>.ObjectCreateCommand command,
+            Map<String, Object> options) throws DBException {
```

Comment:
Formatting, missing annotations
```suggestion
    protected void addObjectCreateActions(
        @NotNull DBRProgressMonitor monitor,
        @NotNull DBCExecutionContext executionContext,
        @NotNull List<DBEPersistAction> actions,
        @NotNull SQLObjectEditor<GenericTableColumn, GenericTableBase>.ObjectCreateCommand command,
        @NotNull Map<String, Object> options
    ) throws DBException {
```